src/loaders/lottie/tvgLottieBuilder.cpp

```diff
@@ -976,6 +976,188 @@ void LottieBuilder::updateImage(LottieGroup* layer)
 }
 
 
+static float _pathLength(const PathCommand* cmds, uint32_t cmdsCnt, const Point* pts, uint32_t ptsCnt)
```

Comment:
this fun is also used in https://github.com/thorvg/thorvg/pull/3151 so after one of the prs is merged I'll reuse it in the other one

Reply:
moved to `tvgMath`